components/business-adaptors/sap/src/main/java/org/wso2/carbon/transports/sap/SAPTransportSender.java

```diff
@@ -293,6 +296,7 @@ private void logon(MessageContext messageContext, JCoDestination destination, St
         if (log.isDebugEnabled()) {
             log.debug("BAPI XMI Logon response: " + logonResponse);
         }
+        log.info("logged in");
```

Comment:
Should this be an info log? It feels like a debug log to me.

Reply:
yes, I think we need to revisit all the log statements, there are multiple places we need to change info logs into debug

Reply:
done